numpy/distutils/system_info.py

```diff
@@ -1580,7 +1580,7 @@ def get_atlas_version(**config):
             log.info('Status: %d', s)
             log.info('Output: %s', o)
 
-    if atlas_version == '3.2.1_pre3.3.6':
+    elif atlas_version == '3.2.1_pre3.3.6':
```

Comment:
Before this change, building locally with a broken compiler would result in the call to `_c_string_literal` below erroring out 
```
  File "/home/matti/pypy_stuff/numpy/numpy/distutils/system_info.py", line 1587, in get_atlas_version
    'ATLAS_INFO', _c_string_literal(atlas_version))
  File "/home/matti/pypy_stuff/numpy/numpy/distutils/system_info.py", line 177, in _c_string_literal
    s = s.replace('\\', r'\\')
AttributeError: 'NoneType' object has no attribute 'replace'

```